client/layout/index.jsx

```diff
@@ -200,14 +201,22 @@ class Layout extends Component {
 	}
 
 	shouldShowOdieAssistant() {
-		// We will only show the Odie Assistant under the "Upgrades" menu.
-		return (
+		// We will only show the Odie Assistant under the "Upgrades" menu. Every automattician should be able to see it under
+		// the feature flag, but we also want to show it to a percentage of users under the Wapuu Sales Agent experiment.
+
+		const isAutomattician =
```

Comment:
I think this constant should be renamed since it doesn't match what it actually contains.